src/DependencyInjection/Configuration.php

```diff
@@ -739,6 +741,27 @@ private function createCachePluginNode()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('cache_listeners')
+                    ->info('An array of classes to act on the response based on the results of the cache check. Must implement '.CacheListener::class.'. Defaults to an empty array.')
```

Comment:
the CachePlugin expects instances. this configuration should be a list of service names rather than class names. a listener might have a non-empty constructor for example.

Reply:
This has been exactly my pov. I replaced it with a list of classes since this lines up with the documentation of the cache plugin.

Reply:
which documentation says it should be class names? the phpdoc on the cache plugin class says its a list of cache listener instances:

https://github.com/php-http/cache-plugin/blob/15883781f0b62b2da6b399675bab9f994a5dd1c7/src/CachePlugin.php#L68

Reply:
Mhmm... didn't find it again... I might have been mistaken. You are right :-)